### PR TITLE
Update dependecies 02/23

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- updated dependencies across monorepo, `react-native` should be checked by @juriczech 
- replaced `dropbox` patch by `@ts-except-error` to save time during updating
- `esbuild` now cannot handle `undefined` env so `NODE_ENV=development` is set for `yarn suite:dev:desktop` 
- added `yarn outdated` plugin, it is more, its more clear than `yarn interactive-tools`
- removed `stringify-object` lib from `connect-explorer` and replaced by `JSON.stringify`

## Related Issue

https://www.notion.so/satoshilabs/Dependency-Management-1b5bf845aa1f4ca7b9d57ea9ccd3fe63?p=a64c0f8e60b54a658f13265f0db777c7&pm=c

#7614 overview of outdated libs excluding `react-native` for now

### CODE REVIEW

**Comment by tomasklim** [comment]
@SocketSecurity ignore core-js@3.27.2
@SocketSecurity ignore core-js-pure@3.27.2
@SocketSecurity ignore nx@15.6.3
@SocketSecurity ignore electron@20.3.12
@SocketSecurity ignore esbuild@0.17.7

regarding the `js-yaml` and `@zkochan/js-yaml` comment here https://github.com/nrwl/nx/pull/11953#issuecomment-1354942089

File: packages/connect-explorer/src/components/Response.tsx [hunk 1]
```diff
@@ -180,11 +179,11 @@ const Response = ({ code, docs, hasDocumentation, response, tab }: ResponseProps
                 switch (tab) {
                     case 'response': {
                         const json = response ? (
-                            <Inspector data={response} expandLevel={10} />
+                            <Inspector data={response} expandLevel={10} table={false} />
```
Comment (dahaca) [inline packages/connect-explorer/src/components/Response.tsx:182]: isn't `table` false [by default](https://github.com/storybookjs/react-inspector#dominspector-)? 🤔

File: packages/connect-explorer/src/reducers/methodReducer.ts [hunk 2]
```diff
@@ -79,16 +77,10 @@ const getParam = (field: Field<any>, $params: Record<string, any> = {}) => {
...
-                      if (Object.prototype.toString.call(obj[prop]) === '[object ArrayBuffer]') {
-                          return 'ArrayBuffer';
-                      }
-                      return originalResult;
-                  },
-              })
+            ? JSON.stringify(state.params, null, 2).replaceAll(
+                  '[object ArrayBuffer]',
+                  'ArrayBuffer',
+              )
```
Comment (dahaca) [inline packages/connect-explorer/src/reducers/methodReducer.ts]:
```suggestion
            ? JSON.stringify(
                    state.params,
                    (key, value) => {
                        if (Object.prototype.toString.call(value) === '[object ArrayBuffer]') {
                            return 'ArrayBuffer';
                        }
                        
                        return value;
                    },
                    2,
                );

```

File: package.json [hunk 3]
```diff
@@ -76,64 +76,64 @@
...
-        "typescript": "4.9.3",
+        "typescript": "4.9.5",
         "react-native": "0.70.5",
-        "prettier": "2.8.0",
+        "prettier": "2.8.4",
         "type-fest": "2.12.2",
         "bcrypto": "5.4.0",
         "react": "18.2.0",
-        "electron": "20.3.5",
+        "electron": "20.3.12",
```
Comment (karliatto) [inline package.json:84]: What about updating electron to 22?  https://github.com/trezor/trezor-suite/issues/7172

**Comment by tomasklim** [comment]
@SocketSecurity ignore esbuild@0.17.8
@SocketSecurity ignore @zkochan/js-yaml@0.0.6
@SocketSecurity ignore js-yaml@3.14.1
@SocketSecurity ignore js-yaml@4.1.0

File: package.json (hunk 3 above)
Comment (tomasklim) [reply package.json:84]: https://github.com/trezor/trezor-suite/pull/7626

File: packages/connect-explorer/src/components/Response.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/connect-explorer/src/components/Response.tsx:182]: Screenshot 2023-02-13 at 19 34 18

Comment (tomasklim) [reply packages/connect-explorer/src/components/Response.tsx:182]: probably not anymore

File: packages/connect-explorer/src/reducers/methodReducer.ts (hunk 2 above)
Comment (tomasklim) [reply packages/connect-explorer/src/reducers/methodReducer.ts]: fixed and squashed

**Comment by Nodonisko** [comment]
Can you try to update also react-redux? Seems there are no real breaking changes for us anyway https://github.com/reduxjs/react-redux/releases/tag/v8.0.0

**Review by dahaca — APPROVED** [review]

**Comment by tomasklim** [comment]
@SocketSecurity ignore core-js@3.28.0
@SocketSecurity ignore core-js-pure@3.28.0

**Comment by tomasklim** [comment]
> Can you try to update also react-redux? Seems there are no real breaking changes for us anyway https://github.com/reduxjs/react-redux/releases/tag/v8.0.0

It actually breaks type checking. @dahaca will look into it #4651

**Pull request merged**